### PR TITLE
Add rakshamaloo dino

### PR DESCRIPTION
**Pull request by rakshamaloo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -565,3 +565,7 @@ harshithap2000-dai_deprivation"
 "Ckbhatia_flying_dino"
 
 ![](Ckbhatia_flying_dino.png)
+
+"rakshamaloo_dino"
+
+![](rakshamaloo_dino.png)
```
Comment (hkatzdev) [inline README.md:571]: Your drawing is really cool! However, you called the file `PicsArt_06-23-10.39.18.jpg`, so you have to change `` to ``

Comment (rakshamaloo) [reply README.md:571]: Okay

**Comment by rakshamaloo** [comment]
@hkatzdev Hi, I have made changes

**Comment by hkatzdev** [comment]
Awesome, LGTM!

**Pull request merged**